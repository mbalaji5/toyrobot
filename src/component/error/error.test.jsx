import { render, screen } from "@testing-library/react";
import { Error } from ".";

describe("Should validate Error component", () => {
  it("should validate snapshot", () => {
    const { baseElement } = render(<Error errorMessage="Error message" />);
    expect(baseElement).toMatchSnapshot();
  });
  it("should validate Error message", () => {
    render(<Error errorMessage="Error message" />);

    const element = screen.getByText(/Error message/i);
    expect(element).toBeDefined();
  });
});
