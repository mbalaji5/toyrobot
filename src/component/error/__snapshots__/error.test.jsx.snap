// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should validate Error component should validate snapshot 1`] = `
<body>
  <div>
    <div
      class="errormessage"
    >
      <span>
        Error message
      </span>
    </div>
  </div>
</body>
`;
